20/alpine3.21/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN addgroup -g 1000 node \
     && export GNUPGHOME="$(mktemp -d)" \
     # gpg keys listed at https://github.com/nodejs/node#release-keys
     && for key in \
-      C0D6248439F1D5604AAFFB4021D900FFDB233756 \
+      5BE8A3F6C8A5C01D106C0AD820B1A390B168D356 \
```

Comment:
Can you undo these key changes in the v20 files, so it doesn't cause a rebuild upstream. They'll get bumped the next time there is a release for v20. It's fine to keep the latest in the `trixie` files